### DISCUSSION TITLE
Tabs - Make round border

### DISCUSSION THREAD

**Discussion by nappalm** [chakra-ui/chakra-ui#5178] [General]
How can I get the border to be rounded?
![image](https://user-images.githubusercontent.com/35016608/144110817-f93c21c8-dee0-4651-b3cb-e29ceb5ad070.png)

**Comment by gabsdotco** [comment]
If it's a progress component, chakra provide parts for this component, so you should update the theme style for the progress component like bellow:

```ts
const theme = {
  ...
  components: {
    Progress: {
      baseStyles: {
        track: {
          borderRadius: 'sm',
        },
      },
    },
  },
  ...
}
```

Chakra don't have a documentation for the components parts names, but they should.

**Reply by nappalm** [reply]
@biewxw its Tab component, the image is an example.

**Comment by anubra266** [comment]
Check for guidance here https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/tabs.ts
